docs/self-managed/console-deployment/configuration.md

```diff
@@ -51,11 +51,13 @@ To enable usage collection, configure the parameters described in the next secti
 
 To enable telemetry, the following parameters need to be configured. Camunda will provide you with the customer ID (Camunda Docker username) needed to send telemetry data to Camunda.
 
-| Parameter        | Description                                                                         | Example value   |
-| ---------------- | ----------------------------------------------------------------------------------- | --------------- |
-| `customerId`     | Unique identifier of the customer. This is also a Camunda Docker registry user name | `customername`  |
-| `installationId` | Unique installation ID of the current customer installation                         | `my-deployment` |
-| `telemetry`      | Telemetry config for Console Self-Managed: `disabled`, `online` or `download`       | `online`        |
+| Parameter                            | Description                                                                                                                                               | Example value                 |
+| ------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------- |
+| `customerId`                         | Unique identifier of the customer. This is also a Camunda Docker registry user name                                                                       | `customername`                |
+| `installationId`                     | Unique installation ID of the current customer installation                                                                                               | `my-deployment`               |
+| `telemetry`                          | Telemetry config for Console Self-Managed: `disabled`, `online` or `download`                                                                             | `online`                      |
+| `managed.releases.tags`              | Assign cluster tags to indicate what type of cluster it is. Default tags are `dev`, `stage`, `test` or `prod`, but you can assign any custom tag as well. | `- dev` (list of strings)     |
```

Comment:
I would suggest moving this into a different section that discusses connecting the Orchestration Cluster to be managed in the Console.

Reply:
@theburi not sure if I understand this correctly. The table describes the config params that can be configured. In console nothing can be really managed so far, it's just a reading view. How about you make a proposal for a new section?

Reply:
I find it difficult to understand how to use this property
"managed.releases.tags" or "managed.releases.custom-properties" are not the actual properties I could use.
maybe an example would help here?

Reply:
@theburi added a section with a full example. can you please have another look?